scim-system-compliant-tests/src/test/java/com/sap/scimono/scim/system/tests/E2ECustomAttributesIntegrationTest.java

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright (c) 2016 by SAP Labs Bulgaria, url: http://www.sap.com All rights reserved.
+ *
+ * This software is the confidential and proprietary information of SAP SE, Walldorf. You shall not disclose such Confidential Information and shall
+ * use it only in accordance with the terms of the license agreement you entered into with SAP.
+ *
+ * Created on Dec 16, 2016 by i061675
+ *
+ */
+
+package com.sap.scimono.scim.system.tests;
+
+import com.sap.scimono.client.SCIMResponse;
+import com.sap.scimono.entity.User;
+import com.sap.scimono.entity.base.Extension;
+import com.sap.scimono.entity.schema.Schema;
+import com.sap.scimono.scim.system.tests.util.TestData;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static javax.ws.rs.core.Response.Status.OK;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class E2ECustomAttributesIntegrationTest extends CompliantSCIMSystemIntegrationTest {
```

Comment:
Follow the same naming pattern: E2ECustomAttributesComplianceTests or even better E2ECustomAttributesTests